war/src/main/less/form/codemirror.less

```diff
@@ -1,3 +1,10 @@
+.CodeMirror-scroll {
+  // CodeMirror text boxes may be resized both horizontally and vertically to ensure
+  // users have enough space to write scripts
+  resize: both;
```

Comment:
These lines shouldn't be there, otherwise looks good to me :)

Reply:
Actually - might be able to do this without any .LESS changes.

<img width="610" alt="image" src="https://user-images.githubusercontent.com/43062514/175820995-945910c3-0ad0-4f0d-bee2-82612430fade.png">

In `textarea.js` line 30 the border is set, remove that line should solve the problem 👍